catalog/transformer/catalog-transformer-xml-binding/src/main/resources/result.xsd

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:mc="urn:catalog:metacard" xmlns="urn:catalog:result"
+        xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="urn:catalog:result"
+        elementFormDefault="qualified" attributeFormDefault="unqualified"
+        version="1.0.0">
+
+	<!-- bring in other schemas -->
+    <xsd:import schemaLocation="metacard.xsd" namespace="mc"/>
+
+    <xsd:element name="result">
+		<xsd:complexType>
+			<xsd:sequence>
+				<xsd:element name="relevance" type="xsd:double" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>A "relevance" score (typical range of 0-1, values over 1 are those that have been boosted) is used to specify a sort policy based on relevance.
+						</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element name="distance" type="xsd:double" minOccurs="0">
```

Comment:
We could have a minInclusive of 0 here too.  We could calculate a maxInclusive for this one as well.